packages/mux-uploader/src/mux-uploader-file-select.ts

```diff
@@ -46,7 +46,10 @@ template.innerHTML = /*html*/ `
 `;
 
 class MuxUploaderFileSelectElement extends globalThis.HTMLElement {
-  #filePickerButton: HTMLElement | null | undefined;
+  #filePickerEl: HTMLElement | null | undefined;
+  #uploaderEl: MuxUploaderElement | null | undefined;
+
+  #controller: AbortController | undefined;
```

Comment:
Might be worth naming it `#abortController`. Since we also have an `uploaderController`, it would make it easier not to get confused between the two.